zerver/lib/management.py

```diff
@@ -80,7 +80,7 @@ def get_users(self, options, realm):
 
             # User should pass either user list or all_users flag
             if bool(options["users"]) == all_users:
-                raise CommandError("You can't use both -u/--users and -a/--all-users.")
+                raise CommandError("Please pass either -u/--users or -a/--all-users.")
```

Comment:
OK, let's just use 2 if statement to handle the 2 cases, each with their own error message.  I think that'll be a lot clearer in both the code and error messages.